targets/csharp/source/source/PlayFabGameserverSDK.cs

```diff
@@ -0,0 +1,203 @@
+﻿// GSDK Only has the following supported frameworks for now
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PlayFab
+{
+    public enum GameState
+    {
+        Invalid,
+        Initializing,
+        StandingBy,
+        Active,
+        Terminating,
+        Terminated,
+        Quarentined
+    }
+
+    public enum GameOperation
+    {
+        Invalid,
+        Continue,
+        GetManifest,
+        Quarantine,
+        Active,
+        Terminate,
+        Operation_Count
+    }
+
+    public static class PlayFabGameserverSDK
+    {
+        // These two keys are only available after allocation (once readyForPlayers returns true)
+        public const string SessionCookieKey = "sessionCookie";
+        public const string SessionIdKey = "sessionId";
+        public const string HeartbeatEndpointKey = "gsmsBaseUrl";
```

Comment:
arent' these legacy values?

Reply:
they are, and I can actually name them whatever since we're setting them. I'll name them something better :)

---
In reply to: [255769737](https://github.com/PlayFab/SDKGenerator/pull/398#discussion_r255769737) [](ancestors = 255769737)